webservices/resources/aggregates.py

```diff
@@ -42,7 +42,7 @@ def build_query(self, committee_id=None, **kwargs):
 
 
 @doc(
-    tags=['schedules/schedule_a'],
+    tags=['receipts'],
```

Comment:
I have heard from some users that they don't think a resource exists, such as line item receipts or independent expenditures. I think it is because they don't know the jargon. 

The tags are used for sorting and grouping the endpoints. 

Here, I am re-naming schedule/schedule to receipts in the top bar. I am thinking that will make it easier for people to find what they need.  I would like to get rid of the / / format to just use them as titles. 

We are already using these as titles and not literal url pieces for /financial/

I did add spaces to "independent expenditures" and other two word phrases, which does look weird with the slashes. I would rather remove the slashes but we can hyphenate those if you think that looks better for now.

![screen shot 2016-08-03 at 12 57 07 pm](https://cloud.githubusercontent.com/assets/4406333/17374272/d16a8baa-5979-11e6-9495-7e43fccada59.png)